code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -181,3 +172,56 @@
 	destination.socks = socks
 	destination.socks_color = socks_color
 	destination.jumpsuit_style = jumpsuit_style
+
+/mob/living/carbon/human/proc/get_age()
+	var/obscured = check_obscured_slots()
+	var/skipface = (wear_mask && (wear_mask.flags_inv & HIDEFACE)) || (head && (head.flags_inv & HIDEFACE))
+	if((obscured & ITEM_SLOT_ICLOTHING) && skipface || isipc(src))
+		return ""
+	switch(age)
+		if(70 to INFINITY)
+			return "Geriatric"
+		if(60 to 70)
+			return "Elderly"
+		if(50 to 60)
+			return "Old"
+		if(40 to 50)
+			return "Middle-Aged"
+		if(24 to 40)
+			return "" //not necessary because this is basically the most common age range
```

Comment:
Need to add something to not display the age examine text because currently it spits out something like "they appear to be ."